### DISCUSSION TITLE
How do I remove the gridlines from the Excel file

### DISCUSSION THREAD

**Discussion by amar132** [ClosedXML/ClosedXML#2431] [General]
I am currently working on a POC but the Excel template that I need to create requires me to remove Gridlines from the Excel sheet. Is there a way to achieve this using ClosedXml?

**Comment by jahav** [comment]
```
using var wb = new XLWorkbook();
wb.ShowGridLines = false; // Used for subsequently created sheets
var ws1 = wb.AddWorksheet();
var ws2 = wb.AddWorksheet();
ws2.ShowGridLines = true; // Setting per sheet

```

**Reply by amar132** [reply]
I feel dumb to ask this question. I wonder how did I miss-out that option. Thanks a ton.
